app/controllers/factory_bot/instrumentation/application_controller.rb

```diff
@@ -3,5 +3,33 @@
 module FactoryBot::Instrumentation
   # A base engine application controller.
   class ApplicationController < ActionController::API
+    # Extend our core functionality to support easy authentication logics
+    include ActionController::HttpAuthentication::Basic::ControllerMethods
+    include ActionController::HttpAuthentication::Digest::ControllerMethods
+    include ActionController::HttpAuthentication::Token::ControllerMethods
```

Comment:
Should these two includes be added already, even though they're not in use yet? Or have they been added so that custom logic (beyond Basic auth) can be implemented more easily thanks to `instance_eval`?

Reply:
Yep, the additional helpers just provide simple building blocks for custom logics.